docs/src/estimators_cheatsheet.md

```diff
@@ -0,0 +1,298 @@
+
+### Canonical TMLE
+
+The way ``\hat{\mathbb{P}}`` is modified is by means of a parametric sub-model also known as a fluctuation. It can be shown that for the conditional mean, it is sufficient to fluctuate ``\hat{Q}_{n, Y}`` only once using the following fluctuations:
+
+- ``\hat{Q}_{Y, \epsilon}(W, T) = \hat{Q}_{n, Y}(T, W) + \epsilon \hat{H}(T, W)``, for continuous outcomes ``Y``.
+- ``\hat{Q}_{Y, \epsilon}(W, T) = \frac{1}{1 + e^{-(logit(\hat{Q}_{n, Y}(T, W)) + \epsilon \hat{H}(T, W))}}``, for binary outcomes ``Y``.
+
+where ``\hat{H}(T, W) = \frac{1(T=t)}{\hat{g}_n(W)}`` is known as the clever covariate. The value of ``\epsilon`` is obtained by minimizing the loss ``L`` associated with ``Q_Y``, that is the mean-squared error for continuous outcomes and negative log-likelihood for binary outcomes. This can easily be done via linear and logistic regression respectively.
+
+!!! note
+    For the ATE and AIE, just like the gradient is linear in ``\Psi``, the clever covariate used to fluctuate the initial ``\hat{Q}_{n, Y}`` is as presented in [Average Treatment Effect and Average Interaction Effect?](@ref)
+
+If we denote by ``\epsilon^*`` the value of ``\epsilon`` minimizing the loss, the TMLE is:
+
+```math
+\hat{\Psi}_{n, TMLE} = \Psi(\hat{Q}_{n, Y, \epsilon^*}, \hat{Q}_{n, W})
+```
+
+### CV-TMLE
+
+Using the same notation as the cross-validated One-Step estimator, the fluctuated distribution is obtained by solving:
+
+```math
+\epsilon^* = \underset{\epsilon}{\arg \min} \frac{1}{n} \sum_{k=1}^K \sum_{\{i: k(i) = k\}} L(Y_i, \hat{Q}_{Y, \epsilon}^{-k}(W_i, T_i, Y_i))
+```
+
+where ``\hat{Q}_{Y, \epsilon}`` and ``L`` are the respective fluctuations and loss for continuous and binary outcomes. This leads to a targeted ``\hat{Q}_{n,Y}^{*}`` such that:
+
+```math
+\forall i \in \{1, ..., n\}, \hat{Q}_{n, Y}^{*}(W_i, T_i) = \hat{Q}_{Y, \epsilon^*}^{-k(i)}(W_i, T_i)
+```
+
+That is, the predictions of ``\hat{Q}_{n, Y}^{*}`` for sample ``i`` are based on the out of fold predictions of ``\hat{Q}_{Y}^{-k(i)}`` and the "pooled" fluctuation given by ``\epsilon^*``.
+
+Then, the CV-TMLE is:
+
+```math
+\begin{aligned}
+\hat{\Psi}_{n, CV-TMLE} &= \sum_{k=1}^K \frac{N_k}{n} \Psi(\hat{Q}_{n, Y}^{*}, \hat{Q}_W^{k}) \\
+&= \frac{1}{n} \sum_{k=1}^K \sum_{\{i: k(i) = k\}} \hat{Q}_{n, Y}^{*}(W_i, T_i)
+\end{aligned}
+```
+
+Notice that, while ``\hat{\Psi}_{n, CV-TMLE}`` is not a plugin estimator anymore, it still respects the natural range of the parameter because it is an average of plugin estimators. Also, because ``\hat{Q}_{n,Y}^*`` is based on the entire data, it seems this estimator is still somehow double-dipping.
```

Comment:
What do you mean by the last sentence "Also, because ``\hat{Q}_{n,Y}^*`` is based on the entire data, it seems this estimator is still somehow double-dipping."? Best to either make precise or remove. Happy to discuss this in more detail.

Reply:
This is because the fluctuation is taken over the whole data. When the empirical distribution (over W) is used, it applies this fluctuation and the terms within the sum are not truly independent.

Reply:
I've removed it because this does not have a great added value yet since I can't really comment further at the moment.